lib/converters/openapi3_to_swagger2.js

```diff
@@ -237,7 +237,7 @@ Converter.prototype.convertParameters = function(obj) {
             this.copySchemaXProperties(param);
             if (!param.description) {
                 const schema = this.resolveReference(this.spec, param.schema);
-                if (schema.description) {
+                if (!!schema && schema.description) {
```

Comment:
Why `!!` instead of just `if (schema)`?

Reply:
Just make sure it return the right result, if `if (schema)` could does so, I can change the code .

Reply:
> Why `!!` instead of just `if (schema)`?

`!!` always return a bool value and more faster than use objcet, here is a discuss https://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript